src/components/Banners/UpdateMessage.jsx

```diff
@@ -2,8 +2,8 @@
 
 import React, { Component } from 'react'
 import { translate } from 'cozy-ui/react/I18n'
```

Comment:
why not adding transpiled for the translate too? Or addind a webpack alias directly?

Reply:
Ah, that was a tets, it's not suppoed to be part of the PR. There already is an alias.